src/locales/gb.json

```diff
@@ -68,5 +68,30 @@
             "Currently, in 2014, the Congregation of Basilian Priests is organizing the House for the Novitiate in the city of Santa Martha, as a budding experience, it will surely require immense efforts, but, redundant in the future of fruits for the work of the Kingdom here on Earth, among our People. "
         ],
         "our-work": "From our experience and Basilian charism carry out a new evangelization that fosters life in all its senses promoting a Community in Christ that nurtures and celebrates the liberation, organization, development and participation of people to live here fully in brotherhood the Kingdom of God, now and forever."
-    }
+    },
+    "content-education": {
+        "students": [
+            "The INSA wants the student.",
+            "Get to know and value yourself.",
+            "Value and enjoy your family and provide solutions to family conflicts.",
+            "Be critical and sensitive to the reality around you.",
+            "Develop your personal potential and skills that allow you to insert yourself in the productive sector.",
+            "Be able to live in a community and work in a team.",
```

Comment:
work as a team? don't you think?

Reply:
I think it is correct.  work in a team... but you are native so do I need to change it as "work as a team"

Reply:
lol, we're gonna have someone who review Englihs as well so no worries, was just wondering